Cargo.toml

```diff
@@ -4,6 +4,7 @@ resolver = "2"
 members = [
     "crates/service_utils",
     "crates/context_aware_config",
+    "crates/cac_demo_app",
```

Comment:
@knutties can we put this in a folder called examples? Similar thing here
https://github.com/juspay/superposition/pull/88/files